src/test/meson.build

```diff
@@ -451,7 +451,7 @@ get_mocks = find_program('../../utils/get-mocks.py')
 # list of copy targets responsible for copying files of each test
 copy_targets = []
 list_files = find_program('../../utils/list-files.py')
-test_files_regex = '(.*TEST[0-9]+([.]PS1)?$)|(.*TESTS.py$)|(.*[.]match$)|(common.sh)|(asset_list.txt)|(alloc_class_config)|(pool.set)|(setup.sh)'
+test_files_regex = '(.*TEST[0-9]+([.]PS1)?$)|(.*TESTS.py$)|(.*[.]match$)|(common.sh)|(asset_list.txt)|(alloc_class_config)|(.*[.]set$)|(setup.sh)|(unicodetest✭✮)|(maps_.*_(linux|freebsd))'
```

Comment:
This regex is getting out of hand :D, what about splitting it into a few lines with '+='? Additionally, it would let us add some comments to each clause and explain why we do it.